model/src/form/form-metadata/index.ts

```diff
@@ -43,7 +43,7 @@ export const phoneSchema = Joi.string().trim()
 export const emailAddressSchema = Joi.string().email().trim().required()
 export const emailResponseTimeSchema = Joi.string().trim().required()
 export const emailSchema = Joi.object<FormMetadataContactEmail>().keys({
-  address: emailAddressSchema,
+  address: emailAddressSchema.lowercase(),
```

Comment:
Loving all this @mokhld

Is there another ticket for when the email is typed it at file download time?

Presume we need to lower it there too so it matches

https://github.com/DEFRA/forms-designer/blob/f688767033d434978158bb800154bdf5b39f4b18/designer/server/src/routes/file/file.js#L155-L157

Reply:
Thanks for the heads up @colinrotherham 👌 . Haven't actually run the app as don't have the local dev account so I'm probably missing this and other things.

Question though, if we're normalising the string to lowercase upon save, then when we retrieve it, it will already be in lowercase?

Reply:
Sorry ignore me. Reading the file, I'm assuming this is we the user enters their email to download the file 😅

Reply:
Yeah, some people might like entering in **Firstname.Lastname** format 😮